db/db-challenge2/chatwork.sql

```diff
@@ -0,0 +1,64 @@
+
+--usersテーブルの作成
```

Comment:
```suggestion
-- usersテーブルの作成
```

--の後に半角スペースが無いとコメントと認識されないはずですが動作確認はされましたでしょうか。
もしくはそちらの環境ではエラーにならないでしょうか。